src/resources/files.ts

```diff
@@ -49,7 +51,9 @@ export interface FilesAPI {
    */
   upload: (
     metadata: ExternalFilesMetadata,
-    fileContent?: FileContent
+    fileContent?: FileContent,
+    // tslint:disable-next-line:bool-param-default
+    waitUntilAcknowledged?: boolean
```

Comment:
this is so ugly:(
wonder how we can workaround this
maybe need to open an issue on tslint repo